bentoml/_internal/yatai_client/errors.py

```diff
@@ -0,0 +1,2 @@
+class YataiCliError(Exception):
```

Comment:
rename YataiClientError?  and should we just put this in `bentoml.exceptions`? which makes it "public" to the user, when using our Python API